code-challenges/combination/challenges-14.test.js

```diff
@@ -0,0 +1,230 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named toTitleCase that takes in an array of strings and returns an array of strings with the first character in upper case and the rest as is.
+
+For example, ['apple', 'banana', 'MacGyver'] returns ['Apple', 'Banana', 'MacGyver'].
+------------------------------------------------------------------------------------------------ */
+
+const toTitleCase = (arr) => {
+  // Solution code here...
+  let myArr = [];
+  arr.forEach(function(val){
+      myArr.push(val.charAt(0).toUpperCase() + val.substring(1, val.length));
```

Comment:
If you don't give `substring()` a second parameter it'll automatically go to the end of the string